src/Resource/Repository.js

```diff
@@ -5,7 +5,8 @@ export default {
    * @returns {Promise<HyralResource[]>}
    */
   find(parameterBag) {
-    return this.connector.fetch(this, parameterBag);
+    return this.connector.fetch(this, parameterBag)
+      .then(response => response.data);
```

Comment:
Dat gaat denk ik niet? Ik neem aan dat de responseSerializer al het object aanpast en hier data en paging in zet? Of bevat data weer een object data?

Reply:
Data bevat weer een object data inderdaad. 
Wat de responseSerializer doet is wordt door axios in de data gezet. 
De rest is allemaal request gerelateerd, waar we denk ik niet veel meer mee hoeven.